src/camera/BoundCamera.ts

```diff
@@ -39,12 +39,22 @@ class BoundCamera extends Camera {
     const firstPanelPrev = firstPanel.range.min;
     const lastPanelNext = lastPanel.range.max;
     const panelAreaSize = lastPanelNext - firstPanelPrev;
-    const canSetBoundMode = viewportSize < panelAreaSize;
+    const isBiggerThanViewport = viewportSize < panelAreaSize;
 
-    if (canSetBoundMode) {
-      this._range = { min: firstPanelPrev + alignPos, max: lastPanelNext - viewportSize + alignPos };
+    const firstPos = firstPanelPrev + alignPos;
+    const lastPos = lastPanelNext - viewportSize + alignPos;
+
+    if (isBiggerThanViewport) {
+      this._range = { min: firstPos, max: lastPos };
     } else {
-      this._range = { min: firstPanel.position, max: lastPanel.position };
+      const align = this._align;
+      const alignVal = typeof align === "object"
+        ? (align as { camera: string | number }).camera
+        : align;
+
+      const pos = firstPos + parseAlign(alignVal, lastPos - firstPos);
```

Comment:
If there are two panels with a width of 30% and the align is `{ camera: "50%", panel: 0 }`, is it placed in a normal position?

It looks like the panel will go out of the viewport area.

Reply:
![image](https://user-images.githubusercontent.com/26213435/129992412-5014227e-c9bb-4e15-a1ac-40e1c461410b.png)
I've test that, and it looks like the above.